### PR TITLE
Add info about beta test dates for self-paced courses

### PR DESCRIPTION
**Pull request by srpearce**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2708](https://openedx.atlassian.net/browse/DOC-2708)

This PR adds info about dates for beta testers in self-paced courses.

### Date Needed (optional)

ASAP

### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer
finishes and gives :+1:. 
- [ ] Subject matter expert: @awaisdar001 
- [ ] Doc team review (sanity check/copy edit/dev edit): @lamagnifica @catong @pdesjardins 

FYI: Tag anyone else who might be interested in this PR here.

### Testing
- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review
- [ ] Add description to release notes task as a comment
- [ ] Squash commits

**Comment by adampalay** [comment]
@srpearce , the subject matter expert should be @awaisdar001 , who actually fixed this bug :)

**Comment by srpearce** [comment]
Thanks @adampalay - tagged @awaisdar001.

File: en_us/shared/releasing_course/beta_testing.rst [hunk 1]
```diff
@@ -14,10 +14,10 @@ This topic describes how to beta test your course.
 Overview
 ****************************
 
-When you are designing a course in Studio, you can select **View Live** to
-test your work work, find gaps and errors, and mimic the learner
-experience. To gather information about the overall experience of taking a
-course, you can also conduct a beta test of the course.
+When you are designing a course in Studio, you can select **View Live** to test
+your work work, find gaps and errors, and mimic the learner experience. To
```
Comment (pdesjardins) [inline en_us/shared/releasing_course/beta_testing.rst]: Not part of your change: "work" is typed twice.

**Comment by pdesjardins** [comment]
Looks good! :+1:

File: en_us/shared/releasing_course/beta_testing.rst [hunk 2]
```diff
@@ -152,14 +152,20 @@ How Beta Testers See Course Content
 
 To beta test a course, you:
 
-* Define a number of days before section and subsection release days for
-  testing to begin.
+* Define a number of days before content releases for testing to begin.
```
Comment (awaisdar001) [inline en_us/shared/releasing_course/beta_testing.rst:155]: Can we mention where to define `number of days`? I mean where the settings exits, just to be clarity.

File: en_us/shared/releasing_course/beta_testing.rst [hunk 3]
```diff
@@ -152,14 +152,20 @@ How Beta Testers See Course Content
...
+
+    * For instructor-paced courses, this number applies to the release date for
+      each section and subsection. Beta testers can only access course content
+      the number of days you specify before it is released.
+
+    * For self-paced courses, this number applies to the course start date.
+      Beta testers can access all course content after the beta test begins.
```
Comment (awaisdar001) [inline en_us/shared/releasing_course/beta_testing.rst]: Just to be clear, `number of days` condition also implies for a `self_paced` course. 
I mean `Beta testers can access course content the number of days you specify before course start date.`

In self_paced courses, courses have start dates but sections/subsections does not have start dates; so the condition of number of days do apply to self_paced courses as well.

File: en_us/shared/releasing_course/beta_testing.rst [hunk 4]
```diff
@@ -201,14 +207,14 @@ Define When the Beta Test Starts
...
 
 To define the start of the beta test, you use Studio to specify a number of
-days before the release day that is defined for each section and subsection in
-your course.
+days before the course start date for self-paced courses or before the section
+and subsection release dates for instructor-paced courses. For more
+information, see :ref:`How_Testers_See_Course`.
 
-#. In Studio, from the **Settings** menu select **Advanced Settings**.
+#. In Studio, from the **Settings** menu, select **Advanced Settings**.
```
Comment (awaisdar001) [inline en_us/shared/releasing_course/beta_testing.rst:217]: Oh I see it is explained here.

File: en_us/shared/releasing_course/beta_testing.rst (hunk 3 above)
Comment (awaisdar001) [inline en_us/shared/releasing_course/beta_testing.rst]: @adampalay  Not sure I understand `after the beta test begins`? Can you help me understand that?

File: en_us/shared/releasing_course/beta_testing.rst [hunk 5]
```diff
@@ -152,14 +152,20 @@ How Beta Testers See Course Content
 
 To beta test a course, you:
 
-* Define a number of days before section and subsection release days for
-  testing to begin.
+* Define a number of days before content releases for testing to begin.
+
+    * For instructor-paced courses, this number applies to the release date for
+      each section and subsection. Beta testers can only access course content
+      the number of days you specify before it is released.
```
Comment (lamagnifica) [inline en_us/shared/releasing_course/beta_testing.rst]: I'd replace "it" with "before each section or subsection"

File: en_us/shared/releasing_course/beta_testing.rst [hunk 6]
```diff
@@ -152,14 +152,20 @@ How Beta Testers See Course Content
...
+
+    * For self-paced courses, this number applies to the course start date.
+      Beta testers can access all course content after the beta test begins.
 
 * Identify beta testers.
 
 Designated beta testers see course content before students can. The matrix
```
Comment (lamagnifica) [inline en_us/shared/releasing_course/beta_testing.rst]: if you have time, maybe take the opportunity to replace "student" with "learner" in this file.

**Comment by lamagnifica** [comment]
a couple of suggestions. :+1:

**Pull request merged**